packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -128,6 +128,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
         <Link
           className={ css('ms-Breadcrumb-itemLink', styles.itemLink) }
           href={ item.href }
+          aria-current={item.isCurrentItem ? 'page' : null}
```

Comment:
@petdun2519 Good question.... aria 1.1 release was/is supposed to be finalized "any day now" (for the last few years). Even one it does become standardized, we should not rely on any aria 1.1 specific functionality for core scenarios in the short term since it will take years for the completely new functionality to be implemented by browsers and ATs. If the items cause a page navigation, then having the items marked up as links is the correct approach (and having them inside of a navigation region makes sense), see http://www.w3.org/TR/wai-aria/roles#link.